### DISCUSSION TITLE
Is it possible to call external php class?

### DISCUSSION THREAD

**Discussion by ddzobov** [zephir-lang/zephir#2413] [Q&A]
I'm trying to call external class method from webonyx/graphql-php

This code causes error:

```
public function getType()
{
     return \GraphQL\Type\Definition\Type::id();
}
```

```
[ERROR] Class name: GraphQL\Type\Definition\Type does not exist in                                                     
         /projects/qwerty/IDAttribute.zep on line 23
```

Seems like there is only one way to do this (very dirty way):
```
public function getType() -> <\GraphQL\Type\Definition\Type>
{
     return eval("return \GraphQL\Type\Definition\Type::id();");
}
```

**Comment by Jeckerson** [comment]
Yes it is possible, try via https://docs.zephir-lang.com/0.12/en/config#external-dependencies

**Comment by ddzobov** [comment]
> Yes it is possible, try via https://docs.zephir-lang.com/0.12/en/config#external-dependencies

Can you please provide an example, how to declare php classes? Because i see only zep files in documentation

**Comment by ddzobov** [comment]
For example, i want to call Model::create method, but getting error. 

Instead of this i tried to do something like call_user_func([new Model(), “create”]) or call_user_func(“Model::create”)  and it works

**Comment by Bellardia** [comment]
Create is not static, have you tried instantiating the class and calling `->create`?
